frontend/src/pages/Demo.tsx

```
@@ -0,0 +1,108 @@
+                    <td className="border p-2"><span className={confClass(i.confidence)}>{confLabel(i.confidence)}</span></td>
+                  </tr>
+                ))}
+              </tbody>
+            </table>
+          </div>
+          {results.risks.length > 0 && (
+            <div className="flex gap-2 flex-wrap">
+              {results.risks.map((r, i) => <span key={i} className={riskClass(r.type)}>{r.message}</span>)}
+            </div>
+          )}
+          <button onClick={handleDownloadIcs} className="bg-green-600 text-white px-4 py-2 rounded">Download .ics Calendar</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)